image/build-wheel.sh

```diff
@@ -16,3 +16,13 @@ bazel run \
     --define NO_IPOPT=ON \
     --define NO_DREAL=ON \
     //:install -- /opt/drake
+
+cd /opt && \
+    tar --create --gzip --file drakewhl-3.7.tar.gz -- \
```

Comment:
```suggestion
cd /opt
tar czf drakewhl-3.7.tar.gz -- \
```

The script is `-e`; no need to chain commands with `&&`.